scripts/GetClrDbg.ps1

```diff
@@ -0,0 +1,128 @@
+﻿<#
+.SYNOPSIS
+Downloads the given $Version of clrdbg for the given $RuntimeID and installs it to the given $InstallPath
+
+.DESCRIPTION
+The following script will generate a project.json and NuGet.config and use dotnet restore and publish to install clrdbg, the .NET Core Debugger
+
+.PARAMETER Version
+Specifies the version of clrdbg to install. Can be 'latest', VS2015U2, or a specific version string i.e. 14.0.25229-preview-2963841
+
+.PARAMETER RuntimeID
+Specifies the .NET Runtime ID of the clrdbg that will be downloaded. Example: ubuntu.14.04-x64. Defaults to the Runtime ID of the current machine.
+
+.Parameter InstallPath
+Specifies the path where clrdbg will be installed. Defaults to the directory containing this script.
+
+.INPUTS
+None. You cannot pipe inputs to GetClrDbg.
+
+.EXAMPLE
+C:\PS> .\GetClrDbg.ps1 -Version latest -RuntimeID ubuntu.14.04-x64 -InstallPath .\clrdbg
+
+.LINK
+https://github.com/Microsoft/MIEngine
+#>
+
+Param (
+    [Parameter(Mandatory=$true)]
+    [string]$Version,
+    [string]$RuntimeID,
+    [string]$InstallPath = (Split-Path -Path $MyInvocation.MyCommand.Definition)
+)
+
+function GetDotNetRuntimeID() {
+    $ridLine = dotnet --info | findstr "RID"
+    
+    if ([System.String]::IsNullOrEmpty($ridLine)) {
+        throw [System.Exception] "Unable to determine runtime from dotnet --info. Make sure dotnet cli is up to date on this machine"
+    }
+
+    $rid = $ridLine.Split(":")[1].Trim();
+
+    if ([System.String]::IsNullOrEmpty($rid)) {
+        throw [System.Exception] "Unable to determine runtime from dotnet --info. Make sure dotnet cli is up to date on this machine"
+    }
+
+    return $rid
+}
+
+# Produces project.json in the current directory
+function GenerateProjectJson([string] $version, [string]$runtimeID) {
```

Comment:
Since I am just outputting this JSON to a file a not actually manipulating it in powershell at all, I think I'm going to leave it as is. The spacing in the file output is wonky if I do it this way. Additionally the imports section needs to be a JSON array and this blob isn't getting translated like that.